.env.example

```
@@ -1,5 +1,24 @@
+PORT=
+API_URL=""
+FRONTEND_URL=""
+DATABASE_URL=""
+APP_SECRET=
+
+# Tests
+SHADOW_DATABASE_URL=""
+
+# Cryptography Settings
+CRYPTO_SECRET=
+CRYPTO_IV=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CRYPTO_IV key should go before the CRYPTO_SECRET key

(UnorderedKey)